pkg/controller/storagecluster/storagecluster.go

```diff
@@ -1321,6 +1321,9 @@ func getZoneMap(nodeList *v1.NodeList, filterLabelKey string, filterLabelValue s
 			return nil, err
 		}
 	}
+	if len(nodeList.Items) == 0 {
+		return nil, fmt.Errorf("node list is empty")
```

Comment:
can we do this validation at where we're dividing 0? I can see based on some filter key or values, it might be valid to have an empty map returned here.

Reply:
Thanks Jiafeng, I will do that!